packages/core/mesh/network-manager/src/transport/datachannel/rtc-data-channel.ts

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import type node from 'node-datachannel';
+
+export class DataChannel extends EventTarget implements RTCDataChannel {
```

Comment:
Please add a class level comment -- at least where was this copied from (and copyright attribution?)